src/ui/react/src/components/base/button-style.js

```diff
@@ -11,11 +11,15 @@
         // Allows validating props being passed to the component.
         propTypes: {
             /**
-             * The style the button should handle as described by http://docs.ckeditor.com/#!/api/CKEDITOR.style
+             * The style the button should handle. It can be a style object as described by http://docs.ckeditor.com/#!/api/CKEDITOR.style
+             * or a string pointing to an object inside the editor instance configuration
```

Comment:
Ok, I'll add an example tomorrow. You can see the usage in the buttons that are now configured with the string. For example,

```javascript
style: 'coreStyles_bold'
```

will get the style from `editor.config.coreStyles_bold` and

```javascript
style: 'myPlugin.myCustomObject.myCustomStyle'
```

will try to get it from `editor.config.myPlugin.myCustomObject.myCustomStyle`

Do you think that would be clear enough? If so, I'll push that tomorrow morning! :)